backend/app/views/spree/admin/products/_form.html.erb

```diff
@@ -104,22 +104,12 @@
       </div>
 
       <div id="shipping_specs">
-        <div id="shipping_specs_height_field" data-hook="admin_product_form_height" class="field alpha two columns">
-          <%= f.label :height %>
-          <%= f.text_field :height, :size => 4 %>
-        </div>
-        <div id="shipping_specs_width_field" data-hook="admin_product_form_width" class="field omega two columns">
-          <%= f.label :width %>
-          <%= f.text_field :width, :size => 4 %>
-        </div>
-        <div id="shipping_specs_depth_field" data-hook="admin_product_form_depth" class="field alpha two columns">
-          <%= f.label :depth %>
-          <%= f.text_field :depth, :size => 4 %>
-        </div>
-        <div id="shipping_specs_weight_field" data-hook="admin_product_form_weight" class="field omega two columns">
-          <%= f.label :weight %>
-          <%= f.text_field :weight, :size => 4 %>
-        </div>
+        <% [:height, :width, :depth, :weight].each_with_index do |field, index| %>
+          <div id="shipping_specs_<%= field %>_field" data-hook="admin_product_form_<%= field %>" class="field two columns <%= index.even? ? 'alpha' : 'omega' %>">
```

Comment:
I think we can lose the data-hook here. It was designed to be matched by deface overrides, but that won't be useful in this circumstance since deface operates on the template itself.